etc/policy.template.toml

```diff
@@ -0,0 +1,338 @@
+#
+# Independent lifetimes can be set for each record.  An integer value is
+# interpreted as seconds.
+dnskey.signature-lifetime = 1209600 # TODO: "2w"
+cds.signature-lifetime = 1209600
+
+# The amount of time remaining before expiry when signatures will be
+# regenerated.
+#
+# In order to prevent a zone's signatures from appearing invalid, they
+# have to be regenerated before they expire.  That hard limit is set by
+# 'signature-lifetime' above.  This setting controls how long before expiry
+# signatures will be regenerated; it must be less than the 'signature-lifetime'
+# setting.
+#
+# Independent waiting times can be set for each record.  An integer value is
+# interpreted as seconds.
+dnskey.signature-remain-time = 604800 # TODO: "1w"
+cds.signature-remain-time = 604800
+
+# How keys are generated.
+[key-manager.generation]
+
+# Whether to generate CSKs, instead of separate ZSKs and KSKs.
+#
+# A CSK (Combined Signing Key) uses the same cryptographic key material for
+# both the ZSK and KSK of a zone.  Both keys are rolled over simultaneously,
+# and only a single cryptographic key needs to be managed; this can simplify the
+# maintenance of the zone.
+#
+# If this is enabled, Cascade will generate CSKs for the associated zones.
+#
+# TODO: Does this affect anything more than generation?  Does it affect how
+# existing keys are rolled over?
+use-csk = false
+
+# Parameters for the cryptographic key material.
+#
+# DNSSEC supports various cryptographic algorithms for signatures; one must be
+# selected, and for some algorithms, additional parameters are also necessary.
+# The same parameters will be applied to the ZSK and KSK.
+#
+# - 'rsa-sha256[:<bits>]', RSASHA256 (algorithm 8), with a public key size of
+#   '<bits>' (default 2048) bits.
+# - 'rsa-sha512[:<bits>]', RSASHA512 (algorithm 10), with a public key size of
+#   '<bits>' (default 2048) bits.
+# - 'ecdsa-p256-sha256', ECDSAP256SHA256 (algorithm 13).
+# - 'ecdsa-p384-sha384', ECDSAP384SHA384 (algorithm 14).
+# - 'ed25519', ED25519 (algorithm 15).
+# - 'ed448', ED448 (algorithm 16).
+#
+# There are additional algorithms, but many are now considered insecure, and it
+# is recommended or mandated to avoid them.  In addition, RSA keys smaller than
+# 2048 bits are not recommended.
+#
+# TODO: This works with HSM keys too, right?
+parameters = "ecdsa-p256-sha256"
+#parameters = "rsa-sha512:4096"
+
+
+# How zones are signed.
+[signer]
+
+# How SOA serial numbers are generated for signed zones.
+#
+# Supported options:
+# - 'keep': use the same serial number as the unsigned zone.
+# - 'counter': increment the serial number every time.
+# - 'unixtime': use the current Unix time, in seconds.
+# - 'date-counter': format the number as '<YYYY><MM><DD><xx>' in decimal.
+#     '<xx>' is a simple counter to allow up to 100 versions per day.
+serial-policy = "date-counter"
+
+# The offset for generated signature inceptions.
```

Comment:
This only affects signatures generated by using a ZSK, as some apex records have their RRSIGs signed by the KSK and use parameters defined above for this.